bitnami/logstash/README.md

```diff
@@ -101,6 +101,7 @@ The following tables lists the configurable parameters of the Logstash chart and
 | `service.loadBalancerIP`                   | loadBalancerIP if service type is `LoadBalancer`                                                                     | `nil`                                                   |
 | `service.loadBalancerSourceRanges`         | Address that are allowed when service is LoadBalancer                                                                | `[]`                                                    |
 | `service.clusterIP`                        | Static clusterIP or None for headless services                                                                       | `nil`                                                   |
+| `service.externalTrafficPolicy`            | Enable client source IP preservation                                                                                 | `Cluster`                                               |
```

Comment:
The description may lead to confusion, as it seems to be a true/false parameter.

I'd suggest replacing with:
```suggestion
| `service.externalTrafficPolicy`            | Sevice external traffic policy, configure to Local to preserve client source IP when using an external loadBalancer.                                                                                 | `""`                                               |```

Reply:
Edited the description suggestion to keep `Cluster` as the default value.